regression/cbmc-java/package_friendly1/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+KNOWNBUG
```

Comment:
Similarly comment why

Reply:
And add a link to the Jira issue!

Reply:
done (added link to github issue since JIRA does not have a cbmc project)

Reply:
Not done yet!

Reply:
Done